api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -270,47 +270,26 @@ public static String getFileAsString(File file) throws IOException {
 	 * @param inputStream Stream to copy from
 	 * @param outputStream Stream/location to copy to
 	 * @throws IOException thrown if an error occurs during read/write
+	 * @return true if file copy was successful, false if was not
+	 * @should not copy the file when outputStream is null
+	 * @should not copy the file when inputStream is null
+	 * @should copy the file
 	 */
-	public static void copyFile(InputStream inputStream, OutputStream outputStream) throws IOException {
+	public static boolean copyFile(InputStream inputStream, OutputStream outputStream) throws IOException {
```

Comment:
dear @lkukielka changing the method signature was not part of the ticket. this would break compatibility with users of this method so please revert that change.